doc/classes/Object.xml

```diff
@@ -1,33 +1,37 @@
 		[codeblocks]
 		[gdscript]
-		var n = Node2D.new()
-		print("position" in n) # Prints "true".
-		print("other_property" in n) # Prints "false".
+		var node = Node.new()
+		print("name" in node)         # Prints true
+		print("get_parent" in node)   # Prints true
+		print("tree_entered" in node) # Prints true
+		print("unknown" in node)      # Prints false
 		[/gdscript]
 		[csharp]
-		var node = new Node2D();
-		// C# has no direct equivalent to GDScript's `in` operator here, but we
-		// can achieve the same behavior by performing `Get` with a null check.
-		GD.Print(node.Get("position") != null); // Prints "true".
-		GD.Print(node.Get("other_property") != null); // Prints "false".
+		var node = new Node();
+		// C# has no all-in-one equivalent to GDScript's `in` operator,
+		// but we can achieve similar behavior with the following methods:
+		GD.Print(node.Get("name") != null);       // Prints true
+		GD.Print(node.Get("unknown") != null);    // Prints false
+		GD.Print(node.HasMethod("get_parent"));   // Prints true
+		GD.Print(node.HasSignal("tree_entered")); // Prints true
 		[/csharp]
 		[/codeblocks]
-		The [code]in[/code] operator will evaluate to [code]true[/code] as long as the key exists, even if the value is [code]null[/code].
-		Objects also receive notifications. Notifications are a simple way to notify the object about different events, so they can all be handled together. See [method _notification].
-		[b]Note:[/b] Unlike references to a [RefCounted], references to an Object stored in a variable can become invalid without warning. Therefore, it's recommended to use [RefCounted] for data classes instead of [Object].
-		[b]Note:[/b] The [code]script[/code] property is not exposed like most properties, but it does have a setter and getter ([code]set_script()[/code] and [code]get_script()[/code]).
+		Notifications are [int] constants commonly sent and received by objects. For example, on every rendered frame, the [SceneTree] notifies nodes inside the tree with a [constant Node.NOTIFICATION_PROCESS]. The nodes receive it and may call [method Node._process] to update. To make use of notifications, see [method notification] and [method _notification].
+		Lastly, every object can also contain metadata (data about data). [method set_meta] can be useful to store information that the object itself does not depend on. To keep your code clean, making excessive use of metadata is discouraged.
+		[b]Note:[/b] Unlike references to a [RefCounted], references to an Object stored in a variable can become invalid unexpectedly. To check for this, use [method @GlobalScope.is_instance_valid]. It's recommended to inherit from [RefCounted] for classes storing data instead of [Object].
```

Comment:
Saying "unexpectedly" to me implies that it's a random occurrence, and can happen on its own. "To check for this" also doesn't connect to the previous sentence in the way that you want it to connect.

```suggestion
		[b]Note:[/b] Unlike references to a [RefCounted], references to an Object stored in a variable can become invalid without invalidating the variable itself. Because of this you should avoid checking such variables against [code]null[/code]. To reliably check for the Object's validity, use [method @GlobalScope.is_instance_valid]. It's recommended to inherit from [RefCounted] for classes storing data instead of [Object].
```